docs/reference/announcements.md

```diff
@@ -30,9 +30,13 @@ Note that the actual values shown in this screenshot don't correspond to any act
 
 ### Removal of Web Modeler's beta API
 
-As of the 8.5 release, the Web Modeler's beta API has been removed. The API was deprecated in 8.3 and is no longer available in 8.5. Use the [Web Modeler v1 API](/apis-tools/web-modeler-api/index.md) instead.
+The Web Modeler's beta API has been removed. The API was deprecated in 8.3 and is no longer available in 8.5. Use the [Web Modeler v1 API](/apis-tools/web-modeler-api/index.md) instead.
 For a migration guide, see the [Web Modeler API documentation](/apis-tools/web-modeler-api/index.md#migrating-from-beta-to-v1).
 
+### Updated SaaS URLs
+
+The SaaS URL is being shortened from cloud.camunda.io to camunda.io. Both URLs will be available until 8.6, after which point there will be an auto-redirect to the shorter URLs. If you have internal whitelisting or active rules for cloud.camunda.io, please transition them to camunda.io. This change primarily affects Console and Modeler. During signup, users are briefly redirected through Accounts (accounts.cloud.camunda.io), which will also be updated.
+
```

Comment:
❓ As a reader I would now expect that the new URL is available with the release of `8.5` but it is not. Should this somehow be reflected that it is coming later?

Reply:
I've asked Support to send a [message](https://docs.google.com/document/d/1EYDzWjcjOoJce8xe3qd5hcHZug-eLR0sN32uCVay1LY/edit#heading=h.vyxt7fn6tmll) alongside the 8.5 release. I've told them to expect a message from you when it's ready to go out. This should ensure that users get informed appropriately, and I don't see a big risk with backporting this change to 8.5 as long as it goes out within the next few weeks.

Reply:
Should this be updated because of the new [communication plan](https://camunda.slack.com/archives/C06PZTAM1PD/p1712575369915109)?

Reply:
Yes. Could you please take over this PR as part of documentation implementation?